packages/pwa/worker/main.js

```diff
@@ -22,6 +22,9 @@ workbox.setConfig({debug: DEBUG})
 // Place your Workbox route configurations here, eg:
 // workbox.routing.registerRoute(...)
 
+// Never cache dev-server internals.
+workbox.routing.registerRoute(/^http:\/\/localhost:3000\/__mrt/, new workbox.strategies.NetworkOnly())
+
```

Comment:
@vcua-mobify I was able to get the dev-server into a weir state because the SW caches everything by default, including the requests to the new `__/mrt/` endpoints. These mustn't be cached, ever, for the dev-server to work properly.

Side note: I don't think the SW we have here has a good default anyway – it has always been a bit of a strange developer-experience to see it serving stale pages on localhost!

Reply:
Nice catch on this. And thanks for the review!